src/modules/esl-avatar/core/esl-avatar.less

```
@@ -0,0 +1,35 @@
+esl-avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: opacity 0.3s ease-in-out;
+  opacity: 0;
+  border-radius: 50%;
+  background: var(--esl-avatar-background);
+  width: var(--esl-avatar-size, 32px);
+  min-width: var(--esl-avatar-size, 32px);
+  height: var(--esl-avatar-size, 32px);
+  min-height: var(--esl-avatar-size, 32px);
+  overflow: hidden;
+
+  &[with-img] {
+    background: transparent;
+  }
+
+  &[ready] {
+    opacity: 1;
+  }
+
+  .esl-avatar-img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  .esl-avatar-text {
+    text-transform: uppercase;
+    line-height: var(--esl-avatar-text-size, 1em);
+    color: var(--esl-avatar-text-color);
+    font-size: var(--esl-avatar-text-size);
```

Propose to move declarations on esl-avatar tag level
```suggestion
    color: inherit;
    font-size: inherit;
```